xonsh/tools.py

```diff
@@ -521,19 +513,16 @@ def always_false(x):
 
 def ensure_string(x):
     """Returns a string if x is not a string, and x if it already is."""
-    if isinstance(x, string_types):
-        return x
-    else:
-        return str(x)
+    return str(x)
```

Comment:
PyPy also behaves like this. if there was a platform that actually didn't, it could be provided by the `platform` module.
my concern is the overhead of function calls and instance checks that effectively result the same like a `str(x)`. the function was used quiet often.

btw, when @GuillaumeLeclerc's Docker script is merged i will submit one to test against different interpreters (it has proven while developing `cerberus`). do you have any othe on mind than CPython 3.4 & 3.5? PyPy3 implements only 3.0 oder 3.2, so that would mean backporting.